modules/system/console/OctoberEnv.php

```diff
@@ -57,6 +58,16 @@ public function fire()
     }
 
     /**
+     * Get the console command options.
+     */
+    protected function getOptions()
+    {
+        return [
+            ['protect', null, InputOption::VALUE_NONE, 'Make sure to replace sensitive config values by the default ones.'],
```

Comment:
Description of option should be "Replaces sensitive config values in the config files with the default values"